integration/benchmark/contLoadGenerator.go

```diff
@@ -60,6 +60,7 @@ type ContLoadGenerator struct {
 	workerStatsTracker  *WorkerStatsTracker
 	workers             []*Worker
 	stopped             bool
+	stoppedChannel      chan bool
```

Comment:
you may want to remove stopped and just use this channel instead.

Also you might want to use `chan struct{}` instead.

Reply:
Swapping to `chan struct{}` but leaving stopped as a separate var for now as replacing it would complicate the check in `Init()` Will leave a TODO in the struct definition for later cleanup.